test/test-clusters/src/main/java/org/elasticsearch/test/cluster/ClusterHandle.java

```diff
@@ -127,4 +128,9 @@ public interface ClusterHandle extends Closeable {
      * Cleans up any resources created by this cluster.
      */
     void close();
+
+    /**
+     * Returns an {@link InputStream} for the given node log.
+     */
+    InputStream streamNodeLog(int index, LogType logType);
```

Comment:
maybe just "getNodeLog"? It's not really "streaming" exactly

Reply:
Done.